src/EditorFeatures/CSharpTest/AddRequiredParentheses/AddRequiredParenthesesTests.cs

```diff
@@ -125,29 +146,16 @@ public async Task TestLogicalPrecedence()
             await TestAsync(
 @"class C
 {
-    void M()
+    void M(bool a, bool b, bool c)
     {
-        int x = a || b $$&& c;
+        bool x = a || b [|&&|] c;
     }
 }",
 @"class C
 {
-    void M()
+    void M(bool a, bool b, bool c)
     {
-        int x = a || (b && c);
-    }
-}", RequireAllParenthesesForClarity);
-        }
-
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddRequiredParentheses)]
-        public async Task TestNoLogicalOnLowerPrecedence()
```

Comment:
Another deleted test?

Reply:
Exact duplicate of `TestLogicalPrecedence`